.github/workflows/docker-builder-devcontainer.yml

```diff
@@ -53,9 +54,12 @@ env:
 jobs:
   build_dockerfile_bazel_base:
     if: |
-      github.event_name == 'push' &&
+      (github.event_name == 'push' &&
```

Comment:
why not just?
```suggestion
      (github.event_name == 'push' || github.event_name == 'workflow_dispatch) &&'
```

Reply:
@maxhbr changes are commited as requested

Reply:
@maxhbr changes are committed as requested